src/components/Navbar/Navbar.scss

```diff
@@ -0,0 +1,38 @@
+.Navbar {
+  width: 100%;
+  height: 50px;
+  align-items: center;
+  display: flex;
+  flex-direction: row;
+  top: 0;
+  font-size: 16px;
+  background-color: var(--color-white);
+  
+  &__left {
+    display: flex;
+    justify-content: flex-start;
+    align-items: center;
+  }
+  
+  &__right {
+    flex: 3;
+    gap: 15px;
+    display: flex;
```

Comment:
let's make gap: 2em; for the sake of responsiveness. 
I dont think you need flex: 3 here as well as justify-content: flex-end;